src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt

```diff
@@ -116,7 +161,8 @@ class StudyService(
         ps.setObject(5, study.lon)
         ps.setObject(6, study.group)
         ps.setObject(7, study.version)
-        ps.setString(8, mapper.writeValueAsString(study.settings))
+        ps.setObject(8, study.contact)
+        ps.setString(9, mapper.writeValueAsString(study.settings))
```

Comment:
Same comment about using a variable for the index. Also, `created_at` is missing.

Reply:
createdAt is not supposed to change and is omitted from being updated.

Reply:
This is for insert.

Reply:
I'm wrong. The postgres default takes care of `created_at`.